src/views/portfolio/projects/Component.vue

```diff
@@ -164,7 +165,10 @@
       syncComponentFields: function(component) {
         this.component = component;
         EventBus.$emit('addCrumb', this.componentLabel);
-      }
+      },
+      redirectToDependencyGraph: function (){
+        this.$router.push({path: "/projects/" + this.component.project.uuid + "/" + this.component.uuid})
```

Comment:
Since in future other paths might be needed, we need a sub-path here. just chaining a second ID will work but is not clear and also might block future use-cases. better: /projects/[ProjectId]/dependencygraph/[componentID]